notebooks/3. Cars.ipynb

```diff
@@ -50,8 +50,10 @@
    "outputs": [],
    "source": [
     "# Select the file to process.\n",
-    "in_file = '../datasets/pointcloud/grown_2386_9702.laz'\n",
-    "out_file = '../datasets/pointcloud/cars_2386_9702.laz'\n",
+    "tilecode = '2386_9702'\n",
+    "\n",
+    "in_file = '../datasets/pointcloud/grown_' + tilecode + '.laz'\n",
```

Comment:
grown is nu niet meer in de volgorde van de notebook naming.

Reply:
Klopt. Misschien gewoon alles "labelled" noemen nu, en die steeds overschrijven? Er is niet echt een reden om al die losse bestanden aan te houden.

Reply:
Ja labelled lijkt me een goeie